### PR TITLE
postgres db configration with microblog

### PR DESCRIPTION
**Pull request by mayuriv-webonise**

### CODE REVIEW

File: app/models/author.rb [hunk 1]
```diff
@@ -0,0 +1,4 @@
+class Author < ApplicationRecord
+    has_many :posts
+    has_many :comments
```
Comment (richa-webonise) [inline app/models/author.rb]: The author and comments would not be having any associations. Please remove it.

File: db/migrate/20230816073715_create_comments.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+class CreateComments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.text :body
+      t.references :post, null: false, foreign_key: true
+      t.references :author, null: false, foreign_key: true
```
Comment (richa-webonise) [inline db/migrate/20230816073715_create_comments.rb]: Author and comments would not be having any associations. Please remove it as discussed.

File: config/database.yml
Comment (richa-webonise) [inline config/database.yml:1]: we don't push the configuration files on GitHub. rather you can add the configurations in the database.yml.example file and push it but avoid pushing any of the credentials on the Git Hub.

Comment (mayuriv-webonise) [reply config/database.yml:1]: ok

File: app/models/author.rb (hunk 1 above)
Comment (mayuriv-webonise) [reply app/models/author.rb]: resolved

File: db/migrate/20230816073715_create_comments.rb (hunk 2 above)
Comment (mayuriv-webonise) [reply db/migrate/20230816073715_create_comments.rb]: removed association

**Review by richa-webonise — APPROVED** [review]

**Pull request merged**